.gitignore

```diff
@@ -0,0 +1,3 @@
+docs/about
+docs/img/logo.png
```

Comment:
Oddly specific ignore...

Reply:
That's because, in this incarnation, we are converting the SVG to PNG for use as the docs logo (it doesn't currently accept SVG, will look to fix or find alternative when we get around to  theming). 

Reference:

https://github.com/jpogran/puppet-vscode/pull/189/files#diff-270eb1b0d54227affb1d5828212609ddR16

Reply:
We probably _don't_ want to ignore **all** future image assets though.